src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationHandler.cs

```diff
@@ -69,9 +69,9 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
             });
 
             using var request = new HttpRequestMessage(HttpMethod.Post, address);
-            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue(System.Net.Mime.MediaTypeNames.Application.Json));
-            using var requestContent = await CopyPayloadAsync("tmp_auth_code", tokens.AccessToken);
-            request.Content = new StringContent(requestContent.RootElement.ToString(), System.Text.Encoding.UTF8, System.Net.Mime.MediaTypeNames.Application.Json);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Json));
+            using var requestContent = new ReadOnlyMemoryContent(await CopyPayloadAsync("tmp_auth_code", "Test"));
```

Comment:
`"Test"`? I guess this should be `tokens.AccessToken`?

Reply:
yes,